workflows/process/artic.nf

```diff
@@ -8,12 +8,38 @@ process artic {
     output:
         tuple val(name), path("*.consensus.fasta"), emit: fasta
         tuple val(name), path("SNP_${name}.pass.vcf"), emit: vcf
-    script:
+    script:   
         """
         artic minion --medaka --medaka-model ${params.medaka_model} --normalise 200 --threads ${task.cpus} --scheme-directory ${external_scheme} \
             --read-file ${reads} nCoV-2019/${params.primerV} ${name}
         zcat ${name}.pass.vcf.gz > SNP_${name}.pass.vcf
 
+        sed -i "1s/.*/>${name}/" *.consensus.fasta
+        """
+}
+
+process artic_nanopolish {
+        label 'artic'
+        publishDir "${params.output}/${params.genomedir}/${name}/", mode: 'copy'
+        publishDir "${params.output}/${params.genomedir}/all_consensi/", mode: 'copy', pattern: "*.consensus.fasta"
+        errorStrategy 'ignore'
+    input:
+        tuple val(name), path(reads), path(external_scheme), path(fast5_dir), path(txt_files)
+    output:
+        tuple val(name), path("*.consensus.fasta"), emit: fasta
+        tuple val(name), path("SNP_${name}.pass.vcf"), emit: vcf
+    script:   
+        """
+        artic minion --minimap2 --normalise 200 \
+            --threads ${task.cpus} \
+            --scheme-directory ${external_scheme} \
+            --read-file ${reads} \
+            --fast5-directory ${fast5_dir} \
+            --sequencing-summary sequencing_summary.txt \
```

Comment:
I think this needs to be `barcoding_summary.txt` in my case?

Reply:
mhhh unsure ill test tomorrow with fast5 with barcodes. but thanks ill test this before merge.

Reply:
@hoelzer can you also show me whats "inside" the workdir?